(SP: 1) Write unit test for "Find offers" page #97

docia opened the pull request:
<img width="880" alt="Screenshot 2024-04-30 at 22 02 24" src="https://github.com/Space2Study-UA-1156/Client-02/assets/102433497/5175a5c8-2d43-4197-873b-0c29c2e81956">

docia committed:
create FindOffers.spec.jsx

Olenka-Hryk commented on tests/unit/components/loader/Loader.spec.jsx:
```diff
@@ -0,0 +1,16 @@
+import Loader from '~/components/loader/Loader.jsx'
```
These changes are already included in this PR #111. Could you please revert then or change the base branch from develop to #111 ?

docia replied:
thanks. fixed!

docia committed:
fix:revert

vlad-khrychov commented on tests/unit/pages/find-offers/FindOffers.spec.jsx:
Mock the `PageWrapper` component. Also, both tests are the same)

docia replied:
mocked PageWrapper and edited one test

docia committed:
update FindOffers.spec.jsx

vlad-khrychov commented on tests/unit/pages/find-offers/FindOffers.spec.jsx:
```diff
@@ -0,0 +1,27 @@
+
+  it('renders the page', () => {
+    render(<FindOffers />)
+  })
```
Look to the sonar messages.
![image](https://github.com/Space2Study-UA-1156/Client-02/assets/67462207/6ff96326-c070-46a1-a1b3-0b523a33dd56)
I think `element= screen.getByTestId('page-wrapper')` would be great.

docia committed:
Merge branch 'develop' into unit-test/#97-find-offers

docia committed:
update FindOffers.spec.jsx

1. updated mock in FindOffers.spec.jsx
2. pulled changes in package-lock. json and test-uttils.jsx

docia committed:
Revert "update FindOffers.spec.jsx"

This reverts commit bff5fa16dc02b84abbe045f8e7d1dcc9084855b9.

docia committed:
Reapply "update FindOffers.spec.jsx"

This reverts commit 23b488cc5cec8f1231a5c4bad0f080399b84d9f7.

docia committed:
fix: correct revert FindOffers.spec.jsx
